src/test/java/com/znsio/sample/e2e/screen/amazon/web/AmazonGridWallPageWeb.java

```diff
@@ -0,0 +1,61 @@
+package com.znsio.sample.e2e.screen.amazon.web;
+
+import com.znsio.e2e.tools.Driver;
+import com.znsio.e2e.tools.Visual;
+import com.znsio.sample.e2e.businessLayer.amazon.GridWallPageBL;
+import com.znsio.sample.e2e.businessLayer.amazon.ProductPageBL;
+import com.znsio.sample.e2e.screen.amazon.AmazonGridWallPageScreen;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class AmazonGridWallPageWeb extends AmazonGridWallPageScreen {
+    private final Visual visually;
+    private final Driver driver;
+    public static final String SCREEN_NAME = AmazonGridWallPageWeb.class.getSimpleName();
+    private static final Logger LOGGER = Logger.getLogger(SCREEN_NAME);
+    private final By header_Results = By.xpath("//span[text()='RESULTS']");
+    private final By list_SearchResults = By.xpath("//div[@data-component-type='s-search-result']");
+
+    public By productFromTheSearchResults(String productToBeSelected) {
+       By productFromTheSearchResults = By.xpath("//div[@data-component-type='s-search-result'][1]//h2//a//span[text()='"+productToBeSelected+"']");
```

Comment:
write better xpaths.. indexes should not be used.

Reply:
Updated the code